sdk/eventhub/event-hubs/README.md

```diff
@@ -181,6 +182,59 @@ for await (const events of consumer.getEventIterator()){
 }
 ```
 
+### Consume events from all partitions of an Event Hub
+
+To consume events for all partitions of an Event Hub, you'll create an `EventProcessor` for a specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get started.
+
+The `EventProcessor` will delegate processing of events to a `PartitionManager` implementation that you provide, allowing your logic to focus on the logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations. In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
+
+```javascript
```

Comment:
Couple of points to simplify the sample here...

- Update `SimplePartitionProcessor` to only implement the required fields i.e we can skip initialize and close
- Skip passing options to `EventProcessor` constructor
- Simplify the factory to a single line  `() => { return new SimplePartitionProcessor() }`
- Skip the const for the factory and use the definition directly in the constructor

Reply:
After the sample, you can add some text like

> To control the number of events passed to `processEvents`, use the options argument in the `EventProcessor` constructor